app/assets/stylesheets/base.css.scss

```
@@ -92,8 +92,4 @@ figcaption {
 .wrap-overflow {
   white-space: normal;
   overflow-wrap: break-word;
-}
-
-
-// Take dark mode into account
-$list-group-bg: $content-bg;
+}
```

Add newline at the end of the file.
```suggestion
}

```